src/components/DandisetStats.vue

```diff
@@ -41,12 +41,8 @@ interface DandisetStats {
 
 export default defineComponent({
   name: 'DandisetStats',
-  setup(props, ctx) {
-    const store = ctx.root.$store;
-
-    const currentDandiset: ComputedRef<Version> = computed(
-      () => store.state.dandiset.publishDandiset,
-    );
+  setup() {
+    const currentDandiset = computed(() => store.state.dandiset.publishDandiset);
```

Comment:
```suggestion
    const currentDandiset = store.state.dandiset.publishDandiset;
```
I'm pretty sure this works and saves some boilerplate.

Reply:
Interesting--where does the reactivity come from?

Reply:
This doesn't seem to work, the variable loses its reactivity if it's not wrapped with `computed`

Reply:
Lame, I assumed that vuex stuff would be reactive, but I guess they don't want direct state mutations?

Reply:
To provide more context, I believe if you were to try to access the state directly in normal `vuex`, it also wouldn't be reactive. This is why calls to `mapState` are always placed in the `computed` section of a standard Vue 2 component. So from that standpoint, this is still a computed var, and all that's changed is how it's declared.

Reply:
Right, the point of VueX is to make state transitions pure and explicit. I'm actually not sure what happens if you try to make an assignment to the state in direct-vuex...

Reply:
Since `direct-vuex` is mostly a type wrapper around `vuex`, I'd expect it to fail in the same way that `vuex` does, by raising an error at runtime, but I'm not actually sure.